rust/psibase/src/main.rs

```diff
@@ -729,7 +729,7 @@ async fn boot(
         progress.inc(1)
     }
     if !args.suppress_ok {
-        println!("Ok");
+        println!("Ok. Successfully booted {}", args.api);
```

Comment:
"Ok" is redundant

Reply:
I actually explicitly left it so that it would be clear in the context of the boot subcommand what the `--suppress-ok` flag would do

Reply:
What about "Ok, booted http://psibase.127.0.0.1.sslip.io:8069/"

Reply:
I'd actually rather get rid of `--suppress-ok` and link the message to the progress bar (`progress.finish_with_message`).

Reply:
Updating the arg and using the ProgressBar can be done separately. For now, I've updated it to "Successfully booted $api"